### ISSUE TITLE
hard refresh of browser is sufficient to load `jupyter_rfb` if it is installed after launching jupyterlab

### ISSUE THREAD

**Issue by kushalkolar** [vispy/jupyter_rfb#101]
Something I've been telling users is that they should restarts their jupyterlab server if they installed `jupyter_rfb` after launching jupyterlab. This isn't actually necessary, a hard refresh is sufficient to get the javascript related stuff loaded in. I can document this, I'm guessing this is just something about how ipywidgets and/or JS works?

**Comment by almarklein** [comment]
>  I'm guessing this is just something about how ipywidgets and/or JS works?

Yes. I don't think I ever restart the server when I am working on jupyter_rfb. When Python code has changed: restart and clear all outputs. When the JS has changed (and is rebuild with yarn): refresh (F5) is enough.
